corehq/ex-submodules/couchexport/util.py

```diff
@@ -175,3 +181,114 @@ def to_json(self, value):
         else:
             function = value
         return function.dumps()
+
+
+def get_excel_format_value(value):
```

Comment:
Would be great to have some tests to verify that various input values for this function produce the expected output format/value.

Reply:
yes! I think this is a great idea. doing this rn

Reply:
added some tests @millerdev 😄